scheduler/tests.py

```diff
@@ -548,9 +550,14 @@ def test_schedule_time_utc(self):
             self.assertEqual(expected, job.schedule_time_utc().isoformat())
 
         def test_unschedulable_old_job(self):
-            job = self.JobClassFactory(scheduled_time=timezone.now() - timedelta(hours=1))
+            job = self.ScheduledJobClassFactory(scheduled_time=timezone.now() - timedelta(hours=1))
             self.assertFalse(job.is_scheduled())
 
+        def test_schedulable_old_job_repeat_none(self):
+            # If repeat is None, the job should be scheduled
+            job = self.RepeatableJobClassFactory(scheduled_time=timezone.now() - timedelta(hours=1), repeat=None)
+            self.assertTrue(job.is_scheduled())
```

Comment:
Added this test to ensure the opposite is Asserts to True.

Reply:
Since this test does not apply to all job-types, it should be in the relavant test-case class.